i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/vue.mdx

```diff
@@ -6,8 +6,8 @@ import Admonition from '@theme/Admonition';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-import AppNote from './components/AppNote';
-import SignInNote from './components/SignInNote';
+import AppNote from './fragments/_app-note.mdx';
+import SignInNote from './fragments/_sign-in-note.mdx';
 
 # 集成 `@logto/vue`
 
```

Comment:
I don't see any change in `android.mdx`. Did you miss it?

Reply:
android didn't use these fragments

Reply:
ok